lib/spack/spack/test/relocate.py

```diff
@@ -276,6 +284,8 @@ def test_set_elf_rpaths(mock_patchelf):
     assert patchelf in output
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
 def test_set_elf_rpaths_warning(mock_patchelf):
```

Comment:
FYI it looks like we have a `@skip_unless_linux` decorator (e.g. the next test), which might be appropriate here since I don't think we'd ever want to be manipulating ELF RPATHs on Windows.

Reply:
Amended.